practice-two/src/components/InputValidate/InputValidate.jsx

```diff
@@ -0,0 +1,54 @@
+import './InputValidate.css';
+import PropTypes from 'prop-types';
+import { memo } from 'react';
+
+const InputValidate = (props) => {
+  const {
+    label,
+    type = 'text',
+    value,
+    errorMessage,
+    genderType,
+    handleChange,
```

Comment:
update name of props not use handle for props function

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/48e6b691-4a41-4a90-a998-1f0d0ce7d15a)